asset/package.json

```
@@ -11,6 +11,7 @@
     "description": "",
     "dependencies": {
         "@terascope/elasticsearch-api": "^2.0.3",
+        "@terascope/teraslice-state-storage": "^0.1.0",
         "@terascope/error-parser": "^1.0.1",
         "@terascope/job-components": "^0.20.0",
```

Can we make sure we are running the latest `job-components`
```suggestion
        "@terascope/job-components": "^0.20.1",
```